test/lib/ufe/test-samples/groupBalls/ballset.ma

```diff
@@ -0,0 +1,189 @@
+//Maya ASCII 2020 scene
```

Comment:
Can you resave this in something like Maya 2019 or 2018 if you have it?  If not, I can grab the file and send an updated version to you (not sure if for this file it's as easy as typing in "2018" by hand).  Matt caught that in one of my recent changes and I installed 2018 so I'd have it around.

Reply:
Sure thing.  Yes just changing the text should do it since we are dealing with the ASCII.  I have been doing it many times before.

The important part to change is this: `requires maya "2020";` 

We have other Maya test scenes that are still 2020. I think we should change them as well ( something for another PR ). I wonder how they could load in older Maya cut since @kxl-adsk usually runs tests against 2019.

Reply:
There is an option to ignore the version, it might be on by default these days?  Changing the requires line usually works, especially in simple files that we create for automated tests that don't have much in them, but it can definitely fail depending on what exactly is in there.

Something else I usually do when creating automated test files is to now save the UI state in the file.  It's usually not needed for the test, it makes the file a lot bigger, and it's something that can mess up interactive tests occasionally.

Reply:
@fowlertADSK please see e123bad7942ab2dd0e392e481f18786c79e9e01a